backend/api/serializers.py

```
@@ -1,8 +1,84 @@
-from django.contrib.auth.models import User
 from rest_framework import serializers
+from .models import (
+    Role, CustomUser, Company, Payment, CoinTransaction,
+    SurveyResponse, Video, Emotion, ShareLink, AnalyticsReport
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)